plugin-modernizer-core/src/main/resources/recipe_data.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Is it something that we need to maintain manually or is is expected to be computed automatically ?

Reply:
right now, to check the working of the function I hard coded it, which will be automated later

Reply:
Sure perfect thanks, for the explanation